uvm/servlets/admin/config/network/MainController.js

```diff
@@ -1092,6 +1092,23 @@ Ext.define('Ung.config.network.MainController', {
                 });
             }
 
+            app = Rpc.directData('rpc.UvmContext.appManager').app('wireguard-vpn');
+            if(app){
+                settings = app.getSettings();
+                settings.tunnels.list.forEach(function(network){
+                    if(network.enabled){
+                        dev = 'tun' + network.id.toString();
```

Comment:
At the moment, all wireguard tunnels go through the wg0 interface.

Reply:
Changed